modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/prepare/partitionawareness/PartitionAwarenessMetadataTest.java

```diff
@@ -158,33 +184,33 @@ public void shortKey(String query, PartitionAwarenessMetadata expected) {
     private static Stream<Arguments> shortKeyMetadata() {
         return Stream.of(
                 // KV GET
-                Arguments.of("SELECT * FROM t WHERE c3=? and c2=?", dynamicParams(0)),
-                Arguments.of("SELECT * FROM t WHERE c2=? and c3=?", dynamicParams(1)),
-                Arguments.of("SELECT * FROM t WHERE c1=? and c2=? and c3=?", dynamicParams(2)),
-                Arguments.of("SELECT * FROM t WHERE c3=? and c1=? and c2=?", dynamicParams(0)),
-                Arguments.of("SELECT * FROM t WHERE c3=? and c2=1", dynamicParams(0)),
-                Arguments.of("SELECT * FROM t WHERE c1=1 and c2=? and c3=?", dynamicParams(1)),
+                Arguments.of("SELECT * FROM t WHERE c3=? and c2=?", meta(0)),
+                Arguments.of("SELECT * FROM t WHERE c2=? and c3=?", meta(1)),
+                Arguments.of("SELECT * FROM t WHERE c1=? and c2=? and c3=?", meta(2)),
+                Arguments.of("SELECT * FROM t WHERE c3=? and c1=? and c2=?", meta(0)),
+                Arguments.of("SELECT * FROM t WHERE c3=? and c2=1", meta(0)),
+                Arguments.of("SELECT * FROM t WHERE c1=1 and c2=? and c3=?", meta(1)),
 
                 Arguments.of("SELECT * FROM t WHERE c3=3", null),
-                Arguments.of("SELECT * FROM t WHERE c2=? and c3=3", null),
-                Arguments.of("SELECT * FROM t WHERE c1=? and c2=? and c3=3", null),
+                Arguments.of("SELECT * FROM t WHERE c2=? and c3=3", meta(new int[]{-1}, new int[]{3})),
+                Arguments.of("SELECT * FROM t WHERE c1=? and c2=? and c3=3", meta(new int[]{-1}, new int[]{3})),
 
                 // KV PUT
-                Arguments.of("INSERT INTO t VALUES (?, ?, ?)",  dynamicParamsTrackingRequired(2)),
-                Arguments.of("INSERT INTO t (c1, c2, c3) VALUES (?, ?, ?)", dynamicParamsTrackingRequired(2)),
-                Arguments.of("INSERT INTO t (c3, c1, c2) VALUES (?, ?, ?)", dynamicParamsTrackingRequired(0)),
+                Arguments.of("INSERT INTO t VALUES (?, ?, ?)",  metaTrackingRequired(2)),
+                Arguments.of("INSERT INTO t (c1, c2, c3) VALUES (?, ?, ?)", metaTrackingRequired(2)),
+                Arguments.of("INSERT INTO t (c3, c1, c2) VALUES (?, ?, ?)", metaTrackingRequired(0)),
 
                 Arguments.of("INSERT INTO t (c1, c2, c3) VALUES (?, ?, 3)", null),
                 Arguments.of("INSERT INTO t (c1, c3, c2) VALUES (?, 3, ?)", null),
                 Arguments.of("INSERT INTO t (c3, c1, c2) VALUES (3, ?, ?)", null),
 
                 // KV DELETE
-                Arguments.of("SELECT * FROM t WHERE c3=? and c2=?", dynamicParams(0)),
-                Arguments.of("SELECT * FROM t WHERE c2=? and c3=?", dynamicParams(1)),
-                Arguments.of("SELECT * FROM t WHERE c3=? and c2=1", dynamicParams(0)),
+                Arguments.of("SELECT * FROM t WHERE c3=? and c2=?", meta(0)),
```

Comment:
these 5 test cases are supposed to be `DELETE` statements, but I forgot to change this copy-past in my PR I merged recently. May you fix this please?